src/openMVG/matching_image_collection/E_ACRobust.hpp

```diff
@@ -102,11 +102,7 @@ struct GeometricFilter_EMatrix_AC
     const std::pair<double,double> ACRansacOut =
       ACRANSAC(kernel, vec_inliers, m_stIteration, &m_E, upper_bound_precision);
 
-#ifdef HAVE_CCTAG
-    if (vec_inliers.size() > KernelType::MINIMUM_SAMPLES)  {
-#else
-    if (vec_inliers.size() > KernelType::MINIMUM_SAMPLES *2.5)  {  
-#endif
+    if (vec_inliers.size() > KernelType::MINIMUM_SAMPLES * OPENMVG_MINIMUM_SAMPLES_COEF)  {  
```

Comment:
i think we should find a better condition as this one can easily go out of the range (ie `MINIMUM_SAMPLES * OPENMVG_MINIMUM_SAMPLES_COEF` > numPoints)

One way to go could be to do an adaptive threshold on the inlier ratio, depending on MINIMUM_SAMPLES. I don't know something like:
```
inlier_ratio > ((k-alpha*MINIMUM_SAMPLES) / MINIMUM_SAMPLES)
```
see http://www.wolframalpha.com/input/?x=0&y=0&i=y%3D(0.15%2B0.05*x)%2Fx+from+x%3D1+to+x%3D8 for an example. This way there is no boundary problem.
Also i think it is meaningful to have higher threshold for models that requires less points (eg a line) than models that requires more points (the fundamental matrix) as the probability of getting a good model decreases. So, eg, if we consider a line, with only two points it's easier to find a good model and hence we want that the estimated model is validated by more points.
What do you think @fabiencastan @lcalvet @pmoulon ?